NDB.Covid19/NDB.Covid19.Droid/Resources/layout/countries_consent.xml

```diff
@@ -44,78 +44,54 @@
                     style="@style/PrimaryTextRegular"/>
             <TextView
-                    android:text="Notification header"
+                    android:text="EU button explanation text"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:paddingTop="@dimen/defaultVerticalMargin"
-                    android:textSize="16sp"
-                    android:layout_below="@id/lookup_text_textview"
-                    android:id="@+id/notification_header_textview"
-                    style="@style/PrimaryTextBold"/>
-            <TextView
-                    android:text="Notification text"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_below="@id/notification_header_textview"
-                    android:id="@+id/notification_text_textview"
+                    android:layout_marginVertical="@dimen/defaultVerticalMargin"
+                    android:layout_below="@id/bodytext2"
+                    android:id="@+id/consent_EU_explanation"
                     style="@style/PrimaryTextRegular"/>
             <TextView
-                    android:text="Be aware text"
+                    android:text="Norway only button explanation text"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:paddingTop="48dp"
-                    android:layout_below="@id/notification_text_textview"
-                    android:id="@+id/beAware_text_textview"
+                    android:layout_below="@id/consent_EU_explanation"
+                    android:id="@+id/consent_onlyNorway_explanation"
                     style="@style/PrimaryTextRegular"/>
-            <TextView
-                    android:text="Notification text"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:paddingVertical="@dimen/defaultVerticalMargin"
-                    android:layout_below="@id/beAware_text_textview"
-                    android:id="@+id/explanation_text_textview"
-                    style="@style/PrimaryTextItalic"/>
             <LinearLayout
                 android:layout_width="fill_parent"
                 android:layout_height="wrap_content"
                 android:orientation="horizontal"
                 android:weightSum="2"
-                android:layout_below="@id/explanation_text_textview">
+                android:layout_below="@id/consent_onlyNorway_explanation">
                 <Button
-                    android:text="Consent Button"
-                    android:layout_marginVertical="@dimen/defaultVerticalMargin"
+                    android:text="Consent Button, EU"
+                    android:layout_marginVertical="36dp"
```

Comment:
are you sure it should have 36 margin? all buttons in the app have the default so it would be the only one that is different

Reply:
Changed to the same size as every other button.